README.md

```diff
@@ -22,31 +20,90 @@ ODK Aggregate can be deployed on Google's App Engine, enabling users to quickly
 * ODK developer Slack archive: [http://opendatakit.slackarchive.io](http://opendatakit.slackarchive.io) 
 * ODK developer wiki: [https://github.com/opendatakit/opendatakit/wiki](https://github.com/opendatakit/opendatakit/wiki)
 
-## Release cycle
+## Getting the code
 
-TBD
+1. Fork the Aggregate project ([why and how to fork](https://help.github.com/articles/fork-a-repo/))
 
-## Setting up your development environment
+1. Clone your fork of the project locally. At the command line:
 
-1. Download and install [Git](https://git-scm.com/downloads) and add it to your PATH
+        git clone https://github.com/YOUR-GITHUB-USERNAME/aggregate
 
-1. Download and install [IntelliJ](https://www.jetbrains.com/idea/) 
+## Setting up the database
 
-1. Fork the Aggregate project ([why and how to fork](https://help.github.com/articles/fork-a-repo/))
+Aggregate supports a variety of DBs, but we strongly recommend you use PostgreSQL first to ensure everything is working. If you wish to use another DB (e.g., Google App Engine, MySQL, or SQLServer databases) after that see [database configurations][docs/database_configurations].
 
-1. Clone your fork of the project locally. At the command line:
+1. Download and install [PostgreSQL 9](https://www.postgresql.org/download) or later
 
-        git clone https://github.com/YOUR-GITHUB-USERNAME/aggregate
+1. Setup your database with these commands
 
-If you prefer not to use the command line, you can use IntelliJ to create a new project from version control using `https://github.com/YOUR-GITHUB-USERNAME/aggregate`. 
+    ```sql
+    CREATE USER "odk_unit" WITH UNENCRYPTED PASSWORD 'test';
+    CREATE DATABASE odk_db WITH OWNER odk_unit;
+    GRANT ALL PRIVILEGES ON DATABASE odk_db TO odk_unit;
+    \connect odk_db;
+    CREATE SCHEMA odk_db;
+    GRANT ALL PRIVILEGES ON SCHEMA odk_db TO odk_unit;
+    ```
```

Comment:
Maybe add a warning telling that these commands have to be executed from the `psql` database shell. Otherwise, the `\connect odk_db` won't work.